Rpi/rollie_pollie.py

```diff
@@ -4,12 +4,17 @@
 from lib.arduino_nfc import SerialNfc
 from lib.scale_observer import ScaleObserver
 from lib.state import State
+from time import sleep
+from Adafruit_CharLCD import Adafruit_CharLCD
 from config import (
     NUMBER_OF_READINGS,
     NFC_PORT,
     CLOCK_PIN, DATA_PIN, TARE_BTN_PIN, REGISTRATION_BTN_PIN,
     CHANNEL, GAIN, SCALE)
-
+# instantiate lcd and specify pins
+lcd = Adafruit_CharLCD(rs=19, en=13,
```

Comment:
Following OOP paradigm; you should instantiate this within the class and reduce the number of variables in the global namespace